config.ts

```diff
@@ -17,6 +17,9 @@ config.google_drive.sheets_folder_ids = [
 
 config.google_drive.assets_folder_ids = ["1abaL1QGd33NqqLoKuo2t9fVWKmh5ouM9", "1kLl5t3o4A2ssC2iC-lT2bp4u8BZZneSE"];
 
+// Exclude uncompressed assets
+config.app_data.assets_filter_function = (fileEntry) => !fileEntry.relativePath.includes("uncompressed")
```

Comment:
I believe this doesn't work because there's already a line below setting the `assets_filter_function` to exclude `draft` folders